### PR TITLE
Add Window and Window Management API

### PR DESCRIPTION
**Pull request by autonome**
A couple notes:

- I moved `alert`, `prompt` and `confirm` to the `<dialog>` feature, which is fiiine, but doesn't feel right. The developer pattern seems to be script-first, not element-first wrt to these features, so rolling these APIs up under `<dialog>` isn't ideal if looking at through Developer Vision. Could separate into two features or leave as-is for now and break it down later.
- A number of keys under `window` are promised to other features already, which I've noted in comments for now. If this PR gets a 👍🏼 I'll remove those comments before merging, but wanted them there for visibility during review, vs looking like a bunch of keys are randomly missing.

### CODE REVIEW

**Comment by autonome** [comment]
>   # baseline: false
>   # support:
>   #   chrome: "1"
>   #   chrome_android: "18"
>   #   edge: "12"
>   #   firefox: "1"
>   #   firefox_android: "4"
>   #   safari: "1"
>   - api.Window.open

hmm need to check this out

**Comment by Elchi3** [comment]
BCD has partial_implementation for window.open https://github.com/mdn/browser-compat-data/blob/85134660ebd65ba1ff2d2561f24fa81ddbbc6619/api/Window.json#L3378

**Comment by autonome** [comment]
@ddbeck I'm reluctant to call this not baseline high for the `window.open` issue. That API has been around since the dawn of the web. Outside of that issue, should be ready for review here.

File: features/dialog.yml [hunk 1]
```diff
@@ -3,3 +3,18 @@ description: The `<dialog>` HTML element represents a modal or non-modal dialog
...
+  - api.HTMLDialogElement.cancel_event
+  - api.HTMLDialogElement.close
+  - api.HTMLDialogElement.close_event
+  - api.HTMLDialogElement.open
+  - api.HTMLDialogElement.returnValue
+  - api.HTMLDialogElement.show
+  - api.HTMLDialogElement.showModal
+  - api.Window.alert
+  - api.Window.confirm
+  - api.Window.prompt
```
Comment (ddbeck) [inline features/dialog.yml:17]: It's unclear to me how these relate to `<dialog>` element (except that they produce modals). I think you should drop these from this PR and pick it up as a separate feature.

File: features/dialog.yml [hunk 2]
```diff
@@ -3,3 +3,18 @@ description: The `<dialog>` HTML element represents a modal or non-modal dialog
...
+  - css.selectors.backdrop.dialog
```
Comment (ddbeck) [inline features/dialog.yml:18]: I think this key belongs in `backdrop.yml`, given that it's the example use-case for that feature.

File: features/history.yml [hunk 3]
```diff
@@ -15,3 +15,5 @@ compat_features:
   - api.History.replaceState
   - api.History.scrollRestoration
   - api.History.state
+  - api.Window.history
```
Comment (ddbeck) [inline features/history.yml:18]: Can you update the description for this feature to mention that this is the entry point?

> "The `window.history` API manipulates the browser session history…

File: features/window-management.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+name: Window Management
```
Comment (ddbeck) [inline features/window-management.yml:1]: [Sentence casing](https://github.com/web-platform-dx/web-features/blob/main/docs/guidelines.md#names), please

```suggestion
name: Window management
```

File: features/window-management.yml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+description: "The Window Management API makes it possible to build multi-screen applications by providing access to information about the screens on a device, and to place content on a specific screen."
```
Comment (ddbeck) [inline features/window-management.yml:2]: More concretely, I think we could say something like:

```suggestion
description: "The `window.getScreenDetails()` method gets information about the screens on a device, which you can use to place new windows on a specific screen with `window.open()`.
```

File: features/window.yml [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+  - api.Window.focus
+  - api.Window.focus_event
+  - api.Window.frameElement
+  - api.Window.frames
+  - api.Window.length
+  - api.Window.load_event
+  - api.Window.name
+  - api.Window.open
+  - api.Window.opener
+  - api.Window.originAgentCluster
```
Comment (ddbeck) [inline features/window.yml:23]: I think this is a separate thing (with `Origin-Agent-Cluster`) that you can drop from this PR.

File: features/window.yml [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+  - api.Window.parent
+  - api.Window.rejectionhandled_event
```
Comment (ddbeck) [inline features/window.yml:25]: I suspect this belongs with `promises`. You can drop it from this PR.

File: features/window.yml [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+  - api.Window.self
+  - api.Window.stop
+  - api.Window.top
+  - api.Window.unhandledrejection_event
```
Comment (ddbeck) [inline features/window.yml:29]: I suspect this belongs with `promises`. You can drop it from this PR.

File: features/window.yml [hunk 9]
```diff
@@ -0,0 +1,30 @@
+name: Window
+description: "The Window object is home to a variety of functions, namespaces, objects, and constructors. Some of these are associated with the user interface window the document is loaded in, but many are items that need to be available in the global scope."
```
Comment (ddbeck) [inline features/window.yml:2]: To create some symmetry with the `navigator` feature you created previously:

```suggestion
description: "The `window` global object represents a browser tab or iframe and its relationship to the document and other tabs. It's also a generic global object, under which many other APIs are located."
```

File: features/window.yml [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+spec: https://html.spec.whatwg.org/multipage/nav-history-apis.html#the-window-object
+status:
+  compute_from: api.Window
+compat_features:
+  - api.Window
+  - api.Window.DOMContentLoaded_event
+  - api.Window.blur_event
+  - api.Window.close
+  - api.Window.closed
+  - api.Window.document
```
Comment (ddbeck) [inline features/window.yml:12]: This could belong in `dom.yml`. Here's fine, but this is yet another case where overlapping keys would be nice.

**Review by ddbeck — CHANGES_REQUESTED** [review]
A few suggestions for you. This is pretty close though.

**Comment by autonome** [comment]
Thanks for review, fixes applied!

**Pull request merged**